src/Authenticators/LDAPMemberLoginHandler.php

```diff
@@ -6,9 +6,10 @@
 use SilverStripe\Control\Email\Email;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Security\Member;
-use SilverStripe\Security\MemberLoginHandler;
+use SilverStripe\Security\MemberAuthenticator\LoginHandler;
+use SilverStripe\Security\MemberAuthenticator\MemberAuthenticator;
```

Comment:
Was this a missed import? I don't see where it's used

Reply:
Yeah unused. Will remove.